packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.scss

```diff
@@ -0,0 +1,30 @@
+@use '../../global.scss';
+
+@mixin text-xl {
+  font-size: 20px !important;
+  line-height: 28px !important;
+}
+@mixin text-sm {
+  font-size: 14px !important;
+  line-height: 20px !important;
+}
+
+.header-text div {
+  @include text-xl;
+  font-family: global.$font-sans;
+  text-align: center !important;
+}
+
+span {
+  color: #0a84ff;
+}
+
+.sync-logo td {
+  width: 240px !important;
+}
+
+.primary-text div {
+  @include text-sm;
+  font-family: global.$font-sans;
+  text-align: center !important;
```

Comment:
If we end up pulling `.primary-text div` into globals to use later, we can probably leave the `text-align` center out of the globals file but leave it here.

Reply:
Yeah we can continue drying up the stylesheets by extending the globals as and when new templates are added